lib/compiler.js

```diff
@@ -11,7 +11,15 @@
  */
 
 const debug = require('debug')('virtual-jade:compiler')
-const assert = require('assert')
+
+function assertWithDetailedError(node, assertion, message) {
+  if (!assertion) {
+    let err = new Error(message)
```

Comment:
side note: you can run `npm run test-ci` locally to run the same suite as Travis